docs/UserGuide.md

```diff
@@ -132,7 +148,7 @@ Examples:
 
 ### Finding persons by name: `find`
 
-Finds persons whose names contain any of the given keywords.
+Finds persons whose names contain any of the given keywords in the displayed person list.
```

Comment:
```suggestion
Finds persons whose names contain any of the given keywords in the _displayed person list_.
```
I think it's bettter to italicize the _displayed person list_.

Reply:
fixed